README.md

```diff
@@ -1,16 +1,132 @@
 # Share
-*This project is under development*
 
-## Project Description
+## Table of Contents
+1. [Introduction](#introduction)
+    1. [About Share](#about-share)
+    2. [Why Share](#why-share)
+2. [Installation and setup](#installation)
+    1. [Install and setting up ODK Collect](#install-collect)
+    2. [Install Share](#install-share)
+3. [Using Share](#using-share)
+    1. [Viewing category of forms](#view-category)
+        1. [View statistics of forms received](#view-statistics)
+        2. [View sent forms](#view-sent-forms)
+        3. [View Received forms](#view-received-forms)
+        4. [Review forms](#review-forms)
+    2. [Send filled forms](#send-filled-forms)
+    3. [Receive filled forms](#receive-filled-forms)
 
-The common requirement of Collect users is to share forms between devices. This app will allow enumerators to share partially/completely filled forms with each other and also help supervisor in reviewing the forms. The main focus of this project is to use different communication methods possible in Android devices so that form can be transferred without any connection issues.
+## 1. Introduction<a name="introduction"></a>
+### 1.1 About Share<a name="about-share"></a>
 
-## Motivation
+Share is an Android application which is a companion app to ODK Collect and it helps different enumerators to share the filled and blanks forms with each other, it also helps organization’s supervisor in reviewing the forms before actually submitting to the server.
 
-There are two use cases which are the main reason for implementing this app
-1. Currently, Collect is used to download blank forms, to fill them and to upload them to the server, there is no collaboration feature yet which help different enumerators to fill the same form with different sections.
-1. Sometimes organizations want to have some review of the forms before sending them to the server. Forms and instances from Collect instead of sending to the server will be sent to this app for the review. Forms from different enumerators of a team will be aggregated to a single device for review.
+### 1.2 Why Share<a name="why-share"></a>
+
+
+Share is designed to enable offline transfer of blank and filled forms. There are three major use cases for this multi-part surveys, supervisor review and blank form distribution.
```

Comment:
There should be a colon to introduce the list of use cases (for this: multi-part)